index.js

```diff
@@ -71,9 +77,38 @@ class App extends MatrixPuppetBridgeBase {
   }
   sendReadReceiptAsPuppetToThirdPartyRoomWithId() {
     // no-op for now
+  }
+  
+  sendImageMessageAsPuppetToThirdPartyRoomWithId(id, data) {
+    return download.getTempfile(data.url, { tagFilename: true }).then(({path}) => {
+      const img = path;
+      let fs = require('fs');
```

Comment:
Please move requires up top with the others. I'm not sure if it will cause any problems to have this ran each time the function executes (I expect it's OK), but it's convention to keep requires in one place.